frontend/package.json

```diff
@@ -0,0 +1,50 @@
+{
+  "name": "traffic-stops",
+  "version": "0.1.0",
+  "private": true,
+  "proxy": "http://localhost:8000/",
+  "dependencies": {
+    "@loadable/component": "^5.13.1",
+    "@nivo/bar": "^0.62.0",
+    "@sentry/react": "^5.22.0",
+    "@testing-library/jest-dom": "^4.2.4",
```

Comment:
Why aren't these in the dev dependencies?

Reply:
Github is terrible about not showing what you actually selected.  I am referring to the stuff labeled testing in the `dependencies`

Reply:
You're right, they should've been in devDeps. But actually those are dead deps. I switched to cypress for testing because those other testing libraries were giving me a headache. I'll push up a commit with those removed!